docs/sql-data-sources-json.md

```diff
@@ -94,3 +94,171 @@ SELECT * FROM jsonTable
 </div>
 
 </div>
+
+## Data Source Option
+
+Data source options of JSON can be set via:
+* the `.option`/`.options` methods of `DataFrameReader` or `DataFrameWriter`
+* the `.option`/`.options` methods of `DataStreamReader` or `DataStreamWriter`
+
+<table class="table">
```

Comment:
Could you provide screenshot for this page as well?

Reply:
Sure, I'll add it. Thanks!